strot/browser/plugin.py

```
@@ -48,25 +48,53 @@ async def click_at_point(self, point: Point) -> bool:
         removed = before_selectors - after_selectors
         return len(added) > 0 or len(removed) > 0
 
+    async def click_element(self, selector: str) -> bool:
+        before_selectors = await self.get_selectors_in_view()
+        try:
+            await self._page.locator(selector).click(timeout=5000)
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)